spec/suites/map/MapSpec.js

```diff
@@ -1418,6 +1418,24 @@ describe('Map', () => {
 
 			map.flyTo(dc, 4, {duration: 0.1});
 		});
+
+		it('flyTo should honour maxZoom', function (done) {
+			this.timeout(10000); // This test takes longer than usual due to frames
```

Comment:
Can we speed up the test with setting `animate: false`? `map.flyTo(newCenter, 22, {animate:false});`

Reply:
I'd say that doing so, we'll not pass in the animation code, which is where the unbounded zoom was. :/

Reply:
Yes that is true, sorry I didn't saw that. But please change the duration and remove the timeout. `map.flyTo(newCenter, 22, {animate: true, duration: 0.1});`

Reply:
Done! Thanks for the suggestion :)